packages/cli/src/resource/strings.json

```diff
@@ -28,6 +28,23 @@
         "force": "Force upgrade the project to work with the latest version of Teams Toolkit."
       },
       "success": "Upgrade project successfully."
+    },
+    "doctor": {
+      "account": {
+        "SideLoadingDisabled": "Your Microsoft 365 tenant admin hasn't enabled sideloading permission for your account. You can't install your app to Teams!",
+        "NotSignIn": "No Microsoft 365 account login",
+        "SignInSuccess": "Microsoft 365 Account (%s) is logged in and sideloading permission enabled."
+      },
+      "node" : {
+        "NotFound": "Cannot find Node.js. Go to https://nodejs.org/about/releases/ to install LTS Node.js.",
+        "NotSupported": "Node.js (%s) is not the officially supported version (%s). Your project may continue to work but we recommend to install the supported version.",
+        "Success": "Node.js version (%s) is installed."
+      },
+      "dotnet" : {
+        "NotFound": "Cannot find .NET Core SDK. For the details why .NET SDK is needed, refer to %s.",
+        "NotSupported": ".NET Core SDK. (%s) is not the officially supported version (%s).",
```

Comment:
NotFound: `Cannot find .NET Core SDK. .NET Core SDK is used for developing Teams app in C# using Teams Toolkit for Visual Studio. Visit https://dotnet.microsoft.com/en-us/download to install the LTS version.`